docs/reference/action_list.rst

```diff
@@ -732,72 +772,77 @@ Checkbox range selection
     You can check / uncheck a range of checkboxes by clicking a first one,
     then a second one with shift + click.
 
-Displaying a non-model field
```

Comment:
Just saw that, I think you missunderstand me.

I said that because we have the `displaying a non model field` section, the 
```
**TODO**: * adding custom columns
```
could be removed.

So I wanted to keep all this section ^^'

Reply:
Ohhh OK ;)

Reply:
Added again.